projects/uitdatabank/models/event-image-put.json

```diff
@@ -0,0 +1,26 @@
+{
+  "title": "event.image.put",
+  "type": "object",
+  "description": "Describes the request body to update an image at an [event](./event.json).",
+  "examples": [],
+  "properties": {
+    "description": {
+      "type": "string",
+      "example": "Picture of the last publiq event",
```

Comment:
It is preferred to add examples to the `examples` property (set on the schema as well but an empty list right now) instead of per property. See also https://www.openapis.org/blog/2021/02/16/migrating-from-openapi-3-0-to-3-1-0 ("Use examples not example")